### ISSUE TITLE
Bug in lazy benchmark code

### ISSUE THREAD

**Issue by evaristocuesta** [linkdotnet/tips-and-tricks#5]
There is a bug in the benchmark code. It is always running ```LazyThreadSafetyMode.None``` mode due to using ```isThreadSafe: false``` instead of using ```Option``` property. 

Original code
```csharp
var lazy = new Lazy<MyClass>(() => new MyClass(), isThreadSafe: false);
```

Proposed code
```csharp
var lazy = new Lazy<MyClass>(() => new MyClass(), Option);
```

**Comment by linkdotnet** [comment]
Ahhh yes! Obviously! Perfect that you found that. If you want you can file a PR.

**Comment by linkdotnet** [comment]
Fixed by #6
